### PR TITLE
chore: init frontend project

### PR DESCRIPTION
**Pull request by KubaBaniak**
[Task description](https://app.clickup.com/t/8694wd802)

### CODE REVIEW

File: frontned/src/styles.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { createTheme } from '@mui/material/styles';
```
Comment (rafalciecwierzmoodup) [inline frontned/src/styles.ts]: I would recommend to change the name type to theme.ts cause this is actually what u export from this file

File: frontned/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import HomePage from './pages/HomePage/HomePage';
+import { ThemeProvider } from '@mui/material';
+import theme from './styles';
+
+const router = createBrowserRouter([
```
Comment (rafalciecwierzmoodup) [inline frontned/src/App.tsx]: Create separate file for handling routes -> [Providers]

File: frontned/src/pages/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function HomePage() {
+  return <p>Hello world!</p>;
```
Comment (rafalciecwierzmoodup) [inline frontned/src/pages/HomePage/HomePage.tsx:2]: Hello! 🚀

File: frontned/src/utils/network.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { QueryClient } from '@tanstack/react-query';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      refetchOnReconnect: false,
+      retry: false,
+      staleTime: 5 * 60 * 1000,
```
Comment (rafalciecwierzmoodup) [inline frontned/src/utils/network.ts]: This could be in const to explain what exactly the time is ;)

File: frontned/src/styles.ts (hunk 1 above)
Comment (rafalciecwierzmoodup) [reply frontned/src/styles.ts]: And move it to some separate folder ;) -> [Providers ]

**Review by rafalciecwierzmoodup — APPROVED** [review]

**Review by rad20c — APPROVED** [review]

**Pull request merged**